<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspekcje</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="index0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>13</number>
        </property>
        <property name="rightMargin">
         <number>13</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border:  0px;
	background-color: rgb(238, 238, 238);
	padding: 3px;
	placeholder-text-color:rgb(120,120,120);
}</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Wszyukaj zabytek...</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollAreaMonuments">
          <property name="styleSheet">
           <string notr="true">QScrollBar:vertical {
	border:none;
	background-color: rgb(222, 221, 218);
	width: 10px;

	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

QScrollBar::handle:vertical {
	background-color: rgb(119, 118, 123);
	min-height: 30px;
	border-radius: 5px;
}

QScrollBar::handle:vertical:hover  {
	background-color: rgb(192, 191, 188);
}

QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(222, 221, 218);
	height: 0px;
}

QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(222, 221, 218);
	height: 0px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>538</width>
             <height>384</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="index1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widgetRange" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>5</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="labelTo">
             <property name="text">
              <string>Do</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelFrom">
             <property name="text">
              <string>Od</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateEdit" name="dateFrom">
             <property name="locale">
              <locale language="Polish" country="Poland"/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboRange">
             <item>
              <property name="text">
               <string>Ten miesiąc</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ten tydzień</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Od początku roku</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Własny</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelRange">
             <property name="text">
              <string>Zakres</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDateEdit" name="dateTo">
             <property name="locale">
              <locale language="Polish" country="Poland"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollAreaInspections">
          <property name="styleSheet">
           <string notr="true">QScrollBar:vertical {
	border:none;
	background-color: rgb(222, 221, 218);
	width: 10px;

	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	border-bottom-left-radius: 5px;
	border-bottom-right-radius: 5px;
}

QScrollBar::handle:vertical {
	background-color: rgb(119, 118, 123);
	min-height: 30px;
	border-radius: 5px;
}

QScrollBar::handle:vertical:hover  {
	background-color: rgb(192, 191, 188);
}

QScrollBar::sub-line:vertical {
	border: none;
	background-color: rgb(222, 221, 218);
	height: 0px;
}

QScrollBar::add-line:vertical {
	border: none;
	background-color: rgb(222, 221, 218);
	height: 0px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::NoFrame</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="layoutInspections">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>30</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_inspections">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>289</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="navbar" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color:#3c3c3c
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_index0">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	color: #d4d4d4;
	background-color:#3c3c3c;
	padding: 7px;
}

QPushButton:hover
{
	border: none;
	color: white;
	background-color:red;
}

QPushButton:disabled
{
	border: none;
	
	color: rgb(119, 118, 123);
	background-color: rgb(36, 31, 49);
}</string>
         </property>
         <property name="text">
          <string>Zabytki</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_index1">
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: none;
	color: #d4d4d4;
	background-color:#3c3c3c;
	padding: 7px;
}

QPushButton:hover
{
	border: none;
	color: white;
	background-color:red;
}

QPushButton:disabled
{
	border: none;
	
	color: rgb(119, 118, 123);
	background-color: rgb(36, 31, 49);
}</string>
         </property>
         <property name="text">
          <string>Inspekcje</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionWyjd"/>
   </widget>
   <widget class="QMenu" name="menuRaporty">
    <property name="title">
     <string>Raporty</string>
    </property>
    <addaction name="actionGeneruj"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuRaporty"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionWyjd">
   <property name="text">
    <string>Wyjdź</string>
   </property>
  </action>
  <action name="actionGeneruj">
   <property name="text">
    <string>Pobierz na ten rok</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
